ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java

```diff
@@ -257,7 +257,7 @@ private void prepareRingtoneList(final ArrayList<Integer> types) {
 
         //Add silent item first
         if (isDisplaySilent) {
-            mRingTones.put(getString(R.string.title_silent_list_item), null /* No ringtone */);
+            mRingTones.put(getString(R.string.title_silent_list_item), Uri.parse("") /* No ringtone */);
```

Comment:
Can you use ‘URI.EMPTY’ instead of ‘URI.parse()’?

Reply:
hi...i already changed the code. now using Uri.EMPTY ...